exercise-2/index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Exercise2</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@300;600&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <h2
+      style="
+        margin-left: 130px;
+        padding: 10px 35px 5px 32px;
+        font-family: 'Raleway';
+        font-size: 30px;
+      "
+    >
+      Why LocoNav's Vehicle Tracking System?
+    </h2>
+
+    <div class="first-container">
+      <img
+        src="img/tracking.png"
+        alt="INTELLIGENT VEHICLE TRACKING"
+        style="height: 90px; width: auto"
+      />
+      <p style="font-weight: bolder">INTELLIGENT VEHICLE TRACKING</p>
+      <p class="trans">
+        Whether you own a single vehicle or a <br />
+        <a href="">fleet owner </a>, LocoNav is determined to <br />
+        protect and monitor your asset from <br />
+        theft or harm
+      </p>
+    </div>
+
+    <div class="second-container">
+      <img
+        src="img/lock.png"
+        alt="VECHILE LOCK"
+        style="height: 90px; width: auto"
+      />
+      <p style="font-weight: bolder">VECHILE LOCK</p>
+      <p class="trans">
+        Turn on vehicle locking system from LocoNav's
+        <a href="">mobile app</a> and be assured that your <br />
+        vehicle won’t start without your <br />
+        permission
+      </p>
+    </div>
+
+    <div class="third-container">
+      <img
+        src="img/alert.png"
+        alt="LIVE ALERT"
+        style="height: 90px; width: auto"
+      />
+      <p style="font-weight: bolder">LIVE ALERT</p>
+      <p class="trans">
+        Get live over-speeding alerts,alerts <br />
+        on entry and exit points, monitor <br />
+        idling in real time,etc. Vehicle <br />
+        services and maintenance alerts
+      </p>
+    </div>
+
+    <div class="fourth-container">
+      <img
+        src="img/health.png"
+        alt="VECHILE HEALTH"
+        style="height: 90px; width: auto"
+      />
+      <p style="font-weight: bolder">VEHICLE HEALTH</p>
+      <p class="trans">
+        Get diagnostic, mileage and health <br />
+        report of your vehicle so that you can <br />
+        worry less about your vehicle's well- <br />
+        being and enjoy your time on the road
+      </p>
+    </div>
+
+    <div class="form">
```

Comment:
this design is **completely broken**. You have randomly center-aligned everything. This looks quite different from the design. Pls read and fix